app/dashboard/static/js/build.js

```diff
@@ -62,14 +62,12 @@
         '*': {
             'app/view-boots-all': 'app/view-boots-all.2017.5',
             'app/view-boots-all-jb': 'app/view-boots-all-jb.2017.3.4',
-            'app/view-boots-all-job': 'app/view-boots-all-job.2017.3.4',
```

Comment:
I think this should have been removed in the other commit about the "boots all job".  It's a detail, but if we need to revert one commit later it would cause some issue.

Reply:
it makes sense, updated